app/services/polyline-builder.js

```diff
@@ -0,0 +1,176 @@
+import Service from '@ember/service';
+import { A } from '@ember/array';
+import { computed } from "@ember/object";
+
+/**
+ * Service for converting Geo LineString (flight paths) to an object that can be 
+ * used by polyline layer.  
+ */
+export default Service.extend({
+  
+    init() {
+        this._super(...arguments);
+    },
+  
+    // Convert linestring to object for polyline
+    convertLineString(mexico, antarctica, russia, india) {
+        let polylineArray = A();
+
+        // Add flight paths
+        if (mexico) {
+            let mexicoLineString = this.get('mexicoLineString');
+            polylineArray.pushObject({
+                coordinates: mexicoLineString.geometry.coordinates,
+                flight: mexicoLineString.properties.flight,
+                flightStatus: mexicoLineString.properties.status,
+                invertIcon: mexicoLineString.properties.invertIcon,
+                statusColour: mexicoLineString.properties.statusColour,
+                popupOpen: mexicoLineString.properties.popupOpen,
+            });
+        }
+        if (antarctica) {
+            let antarcticaLineString = this.get('antarcticaLineString');
+            polylineArray.pushObject({
+                coordinates: antarcticaLineString.geometry.coordinates,
+                flight: antarcticaLineString.properties.flight,
+                flightStatus: antarcticaLineString.properties.status,
+                invertIcon: antarcticaLineString.properties.invertIcon,
+                statusColour: antarcticaLineString.properties.statusColour,
+                popupOpen: antarcticaLineString.properties.popupOpen,
+            });
+        }
+        if (russia) {
+            let russiaLineString = this.get('russiaLineString');
+            polylineArray.pushObject({
+                coordinates: russiaLineString.geometry.coordinates,
+                flight: russiaLineString.properties.flight,
+                flightStatus: russiaLineString.properties.status,
+                invertIcon: russiaLineString.properties.invertIcon,
+                statusColour: russiaLineString.properties.statusColour,
+                popupOpen: russiaLineString.properties.popupOpen,
+            });
+        }
+        if (india) {
+            let indiaLineString = this.get('indiaLineString');
+            polylineArray.pushObject({
+                coordinates: indiaLineString.geometry.coordinates,
+                flight: indiaLineString.properties.flight,
+                flightStatus: indiaLineString.properties.status,
+                invertIcon: indiaLineString.properties.invertIcon,
+                statusColour: indiaLineString.properties.statusColour,
+                popupOpen: indiaLineString.properties.popupOpen,
+            });
+        }
+        return polylineArray;
+    },
+
```

Comment:
LineStrings below belong in a database :)

Reply:
As you're about to transfer this in.... have a good think about our desired behaviour and the implementation tasks to get us there.

Reply:
Geo Table i'd say
Will do, obviously these hard-coded co-ords are just for prototyping